### PR TITLE
updated CVE

### PR DESCRIPTION
**Pull request by jerae-duffin**
Updated CVE content and links to release notes

### CODE REVIEW

File: security/index.md [hunk 1]
```diff
@@ -6,6 +6,18 @@ toc_min: 1
 toc_max: 2
 ---
 
+## CVE-2021-45449
+
+Docker Desktop version 4.3.0 and 4.3.1 has a bug that may log sensitive information (access token or password) on the user's machine during login. This only affects users if they are on Docker Desktop 4.3.0, 4.3.1 and the user has logged in while on 4.3.0, 4.3.1. Gaining access to this data would require having access to the user’s local files. This vulnerability has been fixed in version 4.3.2 or higher. Users should update to this version and may want to update their password. Users should not send local log files to anyone. Users can manually delete their log files, they can be located in the following folder: ~/Library/Containers/com.docker.docker/Data/log/host/ on Mac, and in C:\Users\<username>\AppData\Roaming\Docker\log\host\ on Windows. When a user installs 4.3.2 or higher, we will delete their local log files, so there is no risk of leakage after an update.
```
Comment (stephanierifai) [inline security/index.md:11]:
```suggestion
Docker Desktop version 4.3.0 and 4.3.1 has a bug that may log sensitive information (access token or password) on the user's machine during login. This only affects users if they are on Docker Desktop 4.3.0, 4.3.1 and the user has logged in while on 4.3.0, 4.3.1. Gaining access to this data would require having access to the user’s local files. This vulnerability has been fixed in version 4.3.2 or higher. Users should update to this version and may want to update their password. Users should not send local log files to anyone. Users can manually delete their log files, they can be located in the following folder: `~/Library/Containers/com.docker.docker/Data/log/host/` on Mac, and in `C:\Users\<username>\AppData\Roaming\Docker\log\host\` on Windows. When a user installs 4.3.2 or higher, we will delete their local log files, so there is no risk of leakage after an update.
```

Comment (stephanierifai) [reply security/index.md:11]: thought maybe this should be written as a code block WDYT?

File: security/index.md [hunk 2]
```diff
@@ -6,6 +6,18 @@ toc_min: 1
...
+
+Additionally, these logs may be included when users upload diagnostics,, meaning access tokens and passwords might have been shared with Docker. This only affects users if they are on Docker Desktop 4.3.0, 4.3.1, and the user has logged in while on 4.3.0, 4.3.1 and have gone through the process of submitting diagnostics to Docker. Only Docker support Engineers working on an active support case could have access to the diagnostic files, minimizing leakage risk from these files. We have deleted all potentially sensitive diagnostic files from our data storage and will continue to delete diagnostics reported from the affected versions on an ongoing basis.
+
+### Assigning CNA
```
Comment (stephanierifai) [inline security/index.md:15]:
```suggestion
```

File: security/index.md [hunk 3]
```diff
@@ -6,6 +6,18 @@ toc_min: 1
...
+MITRE Corporation
```
Comment (stephanierifai) [inline security/index.md:16]:
```suggestion
```

Comment (stephanierifai) [reply security/index.md:16]: On second thought, I don't think we need this :)

File: security/index.md [hunk 4]
```diff
@@ -6,6 +6,18 @@ toc_min: 1
...
+
+Additionally, these logs may be included when users upload diagnostics,, meaning access tokens and passwords might have been shared with Docker. This only affects users if they are on Docker Desktop 4.3.0, 4.3.1, and the user has logged in while on 4.3.0, 4.3.1 and have gone through the process of submitting diagnostics to Docker. Only Docker support Engineers working on an active support case could have access to the diagnostic files, minimizing leakage risk from these files. We have deleted all potentially sensitive diagnostic files from our data storage and will continue to delete diagnostics reported from the affected versions on an ongoing basis.
```
Comment (stephanierifai) [inline security/index.md:13]:
```suggestion
Additionally, these logs may be included when users upload diagnostics, meaning access tokens and passwords might have been shared with Docker. This only affects users if they are on Docker Desktop 4.3.0, 4.3.1, and the user has logged in while on 4.3.0, 4.3.1 and have gone through the process of submitting diagnostics to Docker. Only Docker support Engineers working on an active support case could have access to the diagnostic files, minimizing leakage risk from these files. We have deleted all potentially sensitive diagnostic files from our data storage and will continue to delete diagnostics reported from the affected versions on an ongoing basis.
View the [cve](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-45449) for continued updates and information.
```

**Review by stephanierifai — COMMENTED** [review]
Looks good, thanks for doing this @jerae-duffin ! Added a few suggestions.

File: security/index.md (hunk 4 above)
Comment (usha-mandya) [inline security/index.md:13]:
```suggestion
Additionally, these logs may be included when users upload diagnostics, meaning access tokens and passwords might have been shared with Docker. This only affects users if they are on Docker Desktop 4.3.0, 4.3.1, and the user has logged in while on 4.3.0, 4.3.1 and have gone through the process of submitting diagnostics to Docker. Only Docker Support Engineers working on an active support case could have access to the diagnostic files, minimizing leakage risk from these files. We have deleted all potentially sensitive diagnostic files from our data storage and will continue to delete diagnostics reported from the affected versions on an ongoing basis.
```

File: security/index.md (hunk 1 above)
Comment (usha-mandya) [reply security/index.md:11]: Yes, that's correct. Thanks Steph

File: security/index.md [hunk 5]
```diff
@@ -6,6 +6,17 @@ toc_min: 1
...
+
+Additionally, these logs may be included when users upload diagnostics, meaning access tokens and passwords might have been shared with Docker. This only affects users if they are on Docker Desktop 4.3.0, 4.3.1, and the user has logged in while on 4.3.0, 4.3.1 and have gone through the process of submitting diagnostics to Docker. Only Docker support Engineers working on an active support case could have access to the diagnostic files, minimizing leakage risk from these files. We have deleted all potentially sensitive diagnostic files from our data storage and will continue to delete diagnostics reported from the affected versions on an ongoing basis.
+View the [cve](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-45449) for continued updates and information.
+
+
+### References
```
Comment (usha-mandya) [inline security/index.md:17]:
```suggestion
### References

```

File: security/index.md [hunk 6]
```diff
@@ -6,6 +6,17 @@ toc_min: 1
...
+* [Release Notes (Windows)](https://docs.docker.com/desktop/windows/release-notes/)
```
Comment (usha-mandya) [inline security/index.md:18]:
```suggestion
* [Release Notes (Windows)](../desktop/windows/release-notes/index.md)
```

File: security/index.md [hunk 7]
```diff
@@ -6,6 +6,17 @@ toc_min: 1
...
+* [Release Notes (Mac)](https://docs.docker.com/desktop/mac/release-notes/)
```
Comment (usha-mandya) [inline security/index.md:19]:
```suggestion
* [Release Notes (Mac)](../desktop/mac/release-notes/index.md)

```

File: security/index.md [hunk 8]
```diff
@@ -6,6 +6,19 @@ toc_min: 1
...
+
+Additionally, these logs may be included when users upload diagnostics, meaning access tokens and passwords might have been shared with Docker. This only affects users if they are on Docker Desktop 4.3.0, 4.3.1, and the user has logged in while on 4.3.0, 4.3.1 and have gone through the process of submitting diagnostics to Docker. Only Docker support Engineers working on an active support case could have access to the diagnostic files, minimizing leakage risk from these files. We have deleted all potentially sensitive diagnostic files from our data storage and will continue to delete diagnostics reported from the affected versions on an ongoing basis.
+View the [cve](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-45449) for continued updates and information.
```
Comment (usha-mandya) [inline security/index.md:14]:
```suggestion
For detailed information, see [CVE-2021-45449](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-45449){: target="_blank" rel="noopener" class="_"}.
```

**Review by usha-mandya — APPROVED** [review]

**Pull request merged**